jsapp/js/editorMixins/editableForm.es6

```diff
@@ -249,12 +257,14 @@ export default assign({
   }, 200),
 
   preventClosingTab() {
+    this.setState({preventNavigatingOut: true});
     $(window).on('beforeunload.noclosetab', function(){
-      return t('you have unsaved changes');
+      return t('You have unsaved changes. Leave form without saving?');
```

Comment:
I agree that this is an improvement, but every time a localized string like this changes, all of the translations have to be re-done (by a human).